mobile/assets/i18n/en-US.json

```diff
@@ -1,4 +1,35 @@
 {
+  "location_permission": "Location permission",
+  "location_permission_content": "Immich needs your permission to access the location permission to read the WiFi name",
```

Comment:
```suggestion
  "location_permission_content": "In order to use the auto-switching feature, Immich needs precise location permission so it can read the current WiFi network's name",
```